src/azul/service/hca_response_v5.py

```diff
@@ -394,6 +394,7 @@ def make_sample(self, sample):
             'cell_lines': ('cellLines', self.make_cell_line),
             'organoids': ('organoids', self.make_organoid),
             'specimens': ('specimens', self.make_specimen),
+            'matrix': ('matrix', lambda x: []),
```

Comment:
```suggestion
            'matrix': ('matrix', lambda _: []),
```

But really, this surprises me. To be discussed in parking lot.